src/__tests__/components/dashboard/shared/HorizontalTimeline.test.js

```diff
@@ -0,0 +1,175 @@
+import Timeline from 'react-calendar-timeline/lib';
+import { fullRenderComponent } from '../../../../utils/testHelpers';
+import HorizontalTimeline from '../../../../components/dashboard/shared/HorizontalTimeline';
+import * as mocks from '../../../../__mocks__/horizontalTimelineMocks';
+
+function setup() {
+  const props = {
+    title: 'this is a title',
+    groups: mocks.groupsMock,
+    items: mocks.itemsMock,
+    legendItems: mocks.legendItemsMock,
+    rangeItems: mocks.rangeItemsMock,
+    defaultRange: '3mo',
+
+    // TO-DO: add these and how to test it
```

Comment:
You can remove these 2 lines. I don't think we need to test whether or not stacking works which is an option in the library we're using.